tools/build.sh

```diff
@@ -3,25 +3,45 @@
 # Exit on first error
 set -e
 
+# Use default C locale to ensure consistent sorting despite the locale or the
+# operating system.
+export LC_ALL=C
+
 output() {
     echo "$(tput bold) $1$(tput sgr0)"
 }
 
 # Check for a working implementation of sha256sum
 SHA256SUM=""
-which sha256sum > /dev/null && SHA256SUM="sha256sum"
+command -v sha256sum > /dev/null && SHA256SUM="sha256sum"
 # For (at least) MacPorts on MacOS, the GNU coreutils have been prefixed with 'g'
 # to be easily seperateable from the BSD ones
-[ -z "${SHA256SUM}" ] && which gsha256sum > /dev/null && SHA256SUM="gsha256sum"
+[ -z "${SHA256SUM}" ] && command -v gsha256sum > /dev/null && SHA256SUM="gsha256sum"
 # Yes, that leaves the possibility that the current version of shasum doesn't
 # support the SHA256 algorithm. But I don't think that's rather common by now...
-[ -z "${SHA256SUM}" ] && which shasum > /dev/null && SHA256SUM="shasum -a 256"
+[ -z "${SHA256SUM}" ] && command -v shasum > /dev/null && SHA256SUM="shasum -a 256"
 
 if [ -z "${SHA256SUM}" ]; then
   output "Please install sha256sum or shasum first."
   exit 1
 fi
 
+# Detecting whether we're using GNU or BSD sed. There is a small but significant
+# idiosyncracy regarding the inplace option (-i).
```

Comment:
`idiosyncracy` has negative connotations. I'd rather use `quirk` or simply `difference` to stay neutral here.

Reply:
Mh, actually, that was the intention and I don't think there is any reason to even think about that here.. That's the common stance regarding the subtle differences between GNU and BSD/UNIX tools. I wouldn't use the phrase "pain in the ass" though  ;-)

Reply:
Hi guys, just my 2 cents as someone who has had to deal with the different versions of the same tools in different platforms: `idiosyncrasies` is indeed often used to describe these differences, take [this stackoverflow answer](https://unix.stackexchange.com/questions/401905/bsd-sed-vs-gnu-sed-and-i?rq=1), for example:

`Macs use the BSD version of utilities such as sed and date, which have their own idiosyncrasies.`

Having said that, I will be the first to admit that we may be [bikeshedding](https://www.urbandictionary.com/define.php?term=bikeshedding) a bit too much here :-).